plugins/Jobs.cpp

```diff
@@ -220,128 +238,192 @@ bool BedrockPlugin_Jobs::processCommand(SQLite& db, BedrockCommand& command) {
         //     Returns:
         //     - jobID - Unique identifier of this job
         //
-        verifyAttributeSize(request, "name", 1, MAX_SIZE_SMALL);
+        // - CreateJobs (jobs)
+        //
+        //     Creates a list of jobs.
+        //
+        //     Parameters:
+        //     - jobs (json array):
+        //          - name  - An arbitrary string identifier (case insensitive)
+        //          - data  - A JSON object describing work to be done (optional)
+        //          - firstRun - A "YYYY-MM-DD HH:MM:SS" datetime of when
+        //                  this job should next execute (optional)
+        //          - repeat - A description of how to repeat (optional)
+        //          - priority - High priorities go first (optional, default 500)
+        //          - unique - if true, it will check that no other job with this name already exists, if it does it will
+        //            return that jobID
+        //          - parentJobID - The ID of the parent job (optional)
+        //
+        //     Returns:
+        //     - jobIDs - array with the unique identifier of the jobs
+        //
 
-        // If unique flag was passed and the job exist in the DB, then we can finish the command without escalating to
-        // master.
-        uint64_t updateJobID = 0;
-        if (request.test("unique")) {
-            SQResult result;
-            SINFO("Unique flag was passed, checking existing job with name " << request["name"]);
-            if (!db.read("SELECT jobID, data "
-                         "FROM jobs "
-                         "WHERE name=" + SQ(request["name"]) + ";",
-                         result)) {
-                throw "502 Select failed";
+        vector<STable> jsonJobs;
+        if (SIEquals(request.methodLine, "CreateJob")) {
+            verifyAttributeSize(request, "name", 1, MAX_SIZE_SMALL);
```

Comment:
Shouldn't this be in peek? and we should ensure we have job names for the CreateJobs command too